prepare-image.sh

```diff
@@ -25,7 +25,8 @@ if [[ $INSTALL_TYPE == "source" ]]; then
     dnf upgrade -y
     # NOTE(dtantsur): pip is a requirement of python3 in CentOS
     dnf install -y python3-pip $BUILD_DEPS
-    pip3 install pip==21.3.1
+    pip3 install "pip"
+    pip3 install "setuptools"
```

Comment:
this is not necessary, we should install setuptools as system package, for example adding python3-setuptools to BUILD_DEPS on L24